packages/features/form-builder/FormBuilderField.tsx

```diff
@@ -311,8 +312,20 @@ export const ComponentForField = ({
     if (!field.optionsInputs) {
       throw new Error("Field optionsInputs is not defined");
     }
+    const moreThanOneInPersonOptions = !!(
+      field.options.filter((field) => {
+        return field.value === DefaultEventLocationTypeEnum.InPerson;
+      }).length > 1
+    );
+
     const options = field.options.map((field) => {
-      return { ...field, value: field.value === "inPerson" ? field.label : field.value };
+      return {
+        ...field,
+        value:
+          field.value === DefaultEventLocationTypeEnum.InPerson && moreThanOneInPersonOptions
+            ? field.label
+            : field.value,
+      };
```

Comment:
FormBuilderField shouldn't know about Locations, it's a generic component to show any kind of FormBuilder field